bitnami/nginx-ingress-controller/README.md

```diff
@@ -216,6 +217,11 @@ This chart includes a `values-production.yaml` file where you can find some para
 - metrics.enabled: false
 + metrics.enabled: true
 ```
+## Notable changes
+
+### 5.3.0
+
+In this version you can indicate the key to download the GeoLite2 databases using the [parameter](#parameters) `maxmindLicenseKey`.
```

Comment:
I don't think this is necessary. We're only including information about important changes when there are upgrade incompatibilities between versions.